app/src/main/res/values/strings.xml

```diff
@@ -70,6 +70,7 @@
     <string name="msg_new_password">Enter New Password</string>
     <string name="msg_confirm_password">Confirm New Password</string>
     <string name="msg_unread_messages">Unread messages</string>
+    <string name="msg_no_chats_available">There are no messages yet in this channel</string>
```

Comment:
There are no messages yet in this channel -> There are no messages in this channel yet.

For Brazilian translation:
Não há mensagens neste canal ainda.